server/chat-commands/info.ts

```diff
@@ -213,8 +213,8 @@ const commands = {
 			if (punishments.length) {
 				buf += `<br />Room punishments: `;
 
-				buf += punishments.map(([room, punishment]) => {
-					const [punishType, punishUserid, expireTime, reason] = punishment;
+				buf += punishments.map(([r, p]) => {
```

Comment:
ditto here (`curRoom`, `punishment`)

Reply:
punishment is a shadowed variable too, any ideas for that

Reply:
`curPunishment`